internal/server/handlers.go

```diff
@@ -39,6 +39,11 @@ func (s *Server) showOrRender(w http.ResponseWriter, r *http.Request) {
 		// of the app the full path to the given location
 		if os.IsNotExist(err) {
 			s.printWarning("attempted to access non-existent path: %s", currentPath)
+			if s.CustomNotFoundPage != "" {
+				fmt.Fprintf(s.LogOutput, "%s %q -- custom %d \n", r.Method, r.RequestURI, http.StatusNotFound)
```

Comment:
This should've been logged by the middleware that logs requests, not sure it needs to be here.

Reply:
```suggestion
```